INSTALL.md

```diff
@@ -161,11 +161,25 @@ In order to create initial tables or perform latest migrations, run:
 podman run -it --network=host quay.io/gevulot/node:latest migrate [--db-url=postgres://<username>:<password>@<dbhost>/gevulot]
 ```
 
-## Gevulot user
+## Gevulot networking
 
-Gevulot node runs as a normal user. For privilege separation purposes, create a user for Gevulot.
```

Comment:
I thought this was already removed. Scrapping it away now as it's not fully correct atm. and further work needs to be done for it.